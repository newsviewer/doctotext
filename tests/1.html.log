Using HTML parser.
Following encoding was detected: utf-8
