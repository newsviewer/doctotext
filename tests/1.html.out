 
_hyperlink_test_ 

parsed links:
http://www.silvercoders.com/ @ index = 2 length = 16
